bindings/python/fdb/impl.py

```diff
@@ -471,7 +471,16 @@ def get_estimated_range_size_bytes(self, begin_key, end_key):
             begin_key, len(begin_key),
             end_key, len(end_key)
             ))
-
+    
+    def get_range_split_points(self, begin_key, end_key, chunkSize):
+        if begin_key is None or end_key is None:
+            raise Exception('Invalid begin key or end key')
```

Comment:
This is a very good point. I am going to change at least `get_estimated_range_size_bytes` to throw an error because both functions, when called against a large range, would be slow so I want the user to be explicit when they hit the whole range in the DB.

I also noticed this behavior is not consistent across different language bindings. I.e. other bindings don't set empty begin or end key to the beginning or end of the normal keys. They don't throw either. Maybe we want to normalize that too?

Reply:
That sounds reasonable, though if you are changing an API that exists in a released version, you'll need to API version it.